.github/workflows/build.yml

```diff
@@ -5,5 +5,3 @@ jobs:
         steps:
             - run: node --version
             - run: npm --version
```

Comment:
Do you really sure we don't we will not gonna use **_in future_**?

Reply:
I dont think we need in future. I will update confluent page as well for reference.